### DISCUSSION TITLE
"Galaxy and Training" paper

### DISCUSSION THREAD

**Discussion by bebatut** [galaxyproject/training-material#1610] [General]
Hi all,

We would like to write a new paper "Galaxy and Training", about anything related to training in Galaxy (training material, workshops, GTN, etc).

We are organizing a call to collect ideas and maybe create a group of people interested in working together on this paper.
To find a slot fitting to the maximum of people, please fill the following doodle (covering October): https://doodle.com/poll/c4uvgzphnxwn9yeq 

For the ones who may not be able to join but still want to contribute, we will take notes and share any document related to this here, so you can still be involved.

**Comment by bebatut** [comment]
Thank you for answering the doodle.

Given the answers, we will meet on [Thursday 10th at 3pm (UTC+2)](https://arewemeetingyet.com/Berlin/2019-10-10/15:00/Galaxy%20&%20Training%20paper) via [Hangout](https://hangouts.google.com/hangouts/_/calendar/Z2FsYXh5dW5pZnJlaWJ1cmdAZ21haWwuY29t.7mnfjkiushjam8ariecf09ic99).
Minutes of the meeting will be taken in [this Google Doc](https://docs.google.com/document/d/15NloO2G9UWl4txGIfU6Y3eVrZEda5KqPFU51ldQkoaQ/edit?usp=sharing). Feel free to drop your ideas there already.

**Comment by bebatut** [comment]
The ideas that came from the meeting can be found in [this Google doc](https://docs.google.com/document/d/15NloO2G9UWl4txGIfU6Y3eVrZEda5KqPFU51ldQkoaQ/edit#heading=h.tfv4vja66w67). Feel free to add any other ideas and your name in the  list of people who want/have time to work on that!

We would like to meet next week. To find a slot next week fitting the maximum of people, please fill the following doodle: https://doodle.com/poll/phv69f8gkpbbbi63

**Comment by bebatut** [comment]
The next meeting will be on [Wednesday 23rd at 4pm (UTC+2)](https://arewemeetingyet.com/Berlin/2019-10-23/16:00/Galaxy%20&%20Training%20paper) via [Zoom](https://zoom.us/j/262846460).
Minutes of the meeting will be taken in [this Google Doc](https://docs.google.com/document/d/15NloO2G9UWl4txGIfU6Y3eVrZEda5KqPFU51ldQkoaQ/edit?usp=sharing)

**Comment by bedroesb** [comment]
@bebatut Hello! I was wondering if the upcoming meeting is on the 4th or the 6th of december, since both dates are mentioned in the [google doc](https://docs.google.com/document/d/15NloO2G9UWl4txGIfU6Y3eVrZEda5KqPFU51ldQkoaQ/edit#).

**Comment by bebatut** [comment]
The next meeting is on the 4th of December
